liquibase-standard/src/main/java/liquibase/database/core/DatabaseUtils.java

```diff
@@ -35,10 +35,9 @@ public static void initializeDatabase(String defaultCatalogName, String defaultS
                     schema = defaultSchemaName;
                 }
                 executor.execute(
-                        new RawSqlStatement("ALTER SESSION SET CURRENT_SCHEMA=" +
-                                database.escapeObjectName(schema, Schema.class)));
+                        new RawParameterizedSqlStatement("ALTER SESSION SET CURRENT_SCHEMA=", database.escapeObjectName(schema, Schema.class)));
```

Comment:
Does this need something like this:

```suggestion
                        new RawParameterizedSqlStatement("ALTER SESSION SET CURRENT_SCHEMA=?", database.escapeObjectName(schema, Schema.class)));
```